modules/src/ics07_tendermint/client_state.rs

```diff
@@ -0,0 +1,82 @@
+use crate::ics02_client::client_type::ClientType;
+use crate::ics23_commitment::CommitmentRoot;
+
+use crate::ics07_tendermint::header::Header;
+use crate::ics24_host::client::ClientId;
+use serde_derive::{Deserialize, Serialize};
+use std::time::Duration;
+use tendermint::lite::Header as liteHeader;
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct ClientState {
+    id: String,
+    trusting_period: Duration,
+    unbonding_period: Duration,
+    latest_header: Header,
```

Comment:
Only commit batched together with the suggestion below (frozen_height):
```suggestion
```

Reply:
fixed